schedule/yast/ext4/ext4@yast-s390x-disk-activation.yaml

```diff
@@ -12,7 +12,7 @@ schedule:
   - installation/access_beta_distribution
   - installation/product_selection/install_SLES
   - installation/licensing/accept_license
-  - installation/disk_activation
+  - installation/configure_dasd
```

Comment:
Done! Btw this filter also includes a zfcp job, but I believe this job's yaml should not include the dasd module, right?

Reply:
when I filter by disk_activation I got all DASD, not sure how you get some zfcp, of course we need to exclude ssh-X because it is not libyui-rest-api.

Reply:
Hmmm now that I check those again it seems I got the zfcp job from somewhere else : P